xmppserver/src/main/java/org/jivesoftware/openfire/session/RemoteSessionTask.java

```diff
@@ -181,7 +181,7 @@ public enum Operation {
          * Operations of outgoing server sessions
          */
         getOutgoingDomainPairs,
-        isUsingServerDialback,
+        getAuthenticationMethod,
```

Comment:
Is this going to cause compatibility issues with mixed version clusters? Does it matter?

Reply:
I suspect it will cause compatibility issues in a cluster. I'm planning to get this in a more-than-patch release (eg, 4.8.0, not 4.7.5). That should lessen the expectancy of version compatibility in a cluster.